### PR TITLE
Add User Profile API and Delivery API support

### PR DESCRIPTION
**Pull request by dzungtran**
- Update latest lib
- Add User Profile API and Delivery API support

### CODE REVIEW

File: src/paymentwall_init.php [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?php
 /**
- * Paymentwall pingback
+ * Paymentwall
  *
  * @package paymentMethod
  * @copyright Copyright 2015 Paymentwall Inc.
- * @version v1.1.1
+ * @version v1.1.2
```
Comment (sevastyanovio) [inline src/paymentwall_init.php]: Let's store versions in releases

File: src/includes/languages/english/modules/payment/paymentwall.php [hunk 2]
```diff
@@ -1,35 +1,12 @@
...
-*/
-
-/**
 * Paymentwall languages
 *
 * @package paymentMethod
-* @copyright Copyright 2014 Paymentwall Inc.
-* @version v1.1.0
+* @copyright Copyright 2015 Paymentwall Inc.
+* @version v1.1.3
```
Comment (sevastyanovio) [inline src/includes/languages/english/modules/payment/paymentwall.php]: Let's store versions in releases

File: src/paymentwall_pingback.php [hunk 3]
```diff
@@ -2,19 +2,40 @@
...
+
 $pingback = new Paymentwall_Pingback($_GET, $_SERVER['REMOTE_ADDR']);
 if ($pingback->validate()) {
+
     $productId = $pingback->getProduct()->getId();
-    $id = tep_db_query("select orders_id from " . TABLE_ORDERS . " where orders_id = '" . (int)$productId . "'");
+    $order = new order($productId);
 
-    if (mysqli_fetch_all($id)[0][0]) {
+    if ($order) {
```
Comment (sevastyanovio) [inline src/paymentwall_pingback.php]: what if product_id will be wrong?

File: src/paymentwall_pingback.php [hunk 4]
```diff
@@ -2,19 +2,42 @@
...
+    $_info = array_filter($order->info);
 
-    if (mysqli_fetch_all($id)[0][0]) {
+    if (!empty($_info)) {
         if ($pingback->isDeliverable()) {
+
+            if (MODULE_PAYMENT_PAYMENTWALL_DELIVERY == 'True') {
+                $delivery = new Paymentwall_GenerericApiObject('delivery');
+                $response = $delivery->post(getDeliveryConfirmationData($order, $pingback->getReferenceId()));
+            }
```
Comment (sevastyanovio) [inline src/paymentwall_pingback.php:21]: Let's add ability to disable this feature from admin area.

Comment (dzungtran) [inline src/paymentwall_pingback.php:21]: This feature can enable or disable.
In OSCommerce, the settings store in the DB and retrieve through constant variables.

**Pull request merged**